cpp/src/io/csv/csv_gpu.cu

```diff
@@ -363,6 +364,14 @@ __inline__ __device__ cudf::dictionary32 decode_value(const char *data,
 {
   return cudf::dictionary32{};
 }
+template <>
+__inline__ __device__ cudf::list_view decode_value(const char *data,
+                                                   long start,
+                                                   long end,
+                                                   ParseOptions const &opts)
+{
+  return cudf::list_view{};
```

Comment:
Is this a TODO? (and the same for the dictionary32 and string_view ones?)

Reply:
Yes.  Although I'm not 100% sure dictionary and list types are something there's even a standard for in csv.

Reply:
Mark with TODO comment?